catalog/spatial/registry/registry-federation-admin-service-impl/src/main/java/org/codice/ddf/registry/federationadmin/service/impl/RefreshRegistryEntries.java

```diff
@@ -68,10 +77,31 @@
 
     private boolean enableDelete = true;
 
+    private boolean initialized = false;
```

Comment:
Instead of using an `initialized` flag, couldn't we just check to see whether `scheduledTask` is `null`?